extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/QuarkusRestPathTemplateInterceptor.java

```diff
@@ -24,7 +24,7 @@
     @AroundInvoke
     Object restMethodInvoke(InvocationContext context) throws Exception {
         QuarkusRestPathTemplate annotation = getAnnotation(context);
-        if (annotation != null) {
+        if ((annotation != null) && (request.getCurrent() != null)) {
```

Comment:
The only thing I'm concerned about is a genuine situation where for some reason `getCurrent()` returns `null` and it should be populated.

Is that possible?

Reply:
Nope, if it happens, it's a bug and way more things will go wrong :)